docs/pages/installation/gke.md

```diff
@@ -6,8 +6,9 @@ permalink: installation/gke
 
 # Quick Start with Google Kubernetes Engine (GKE)
 
-Make your way to Meshery and download the token by clicking the "Get Token" option in the dropdown menu under your User Account avatar.
-<br>Utilize the token to run the following command:
+- Make your way to meshery.ui, login with your user details and head over to the local port, found at <code>localhost:3000</code>.
```

Comment:
"Navigate" more appropriate than "make your way".
"Meshery UI" more appropriate than "meshery.ui"
Avoid run-on sentence by placing a period after "In your web browser, navigate to Meshery UI being sure to include a port number as appropriate (e.g. `http://localhost:9081`)".
"Download an authentication token by..."

Don't use port 3000 as that is the port used for developers/contributors working on Meshery.